src/ProofOfConcepts/ConnectorMessageDrawer/DrawerSplit/ConnectorDrawerMessageStatistics.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { FunctionComponent } from "react";
+
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  CardTitle,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Divider,
+  Flex,
+  FlexItem,
+  Grid,
+  GridItem,
+  Split,
+  SplitItem,
+  Text,
+  TextContent,
+  TextListItem,
+  TextListItemVariants,
+  TextVariants,
+  Title,
+} from "@patternfly/react-core";
+import {
+  global_success_color_100,
+  global_danger_color_100,
+} from "@patternfly/react-tokens";
+
+import { ExclamationIcon } from "@patternfly/react-icons";
+import CheckIcon from "@patternfly/react-icons/dist/esm/icons/check-icon";
+
+import "./ConnectorDrawer.css";
+
+export type ConnectorDrawerMessageStatisticsProps = {
+  sent: string;
+  notSent: string;
+};
+
+export const ConnectorDrawerMessageStatistics: FunctionComponent<
+  ConnectorDrawerMessageStatisticsProps
+> = ({ sent, notSent }) => {
+  return (
+    <Flex direction={{ default: "column" }}>
+      <FlexItem>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h4">Processed messages</Title>
+          </CardTitle>
+          <CardBody>
+            <Split>
```

Comment:
The Split layout supports including a gutter, and we should use that here. 

In theory the gutter size here should be larger than the space between the icon and the text, so that "1600 sent" is never closer to the "!" icon vs the ✅ icon. But this would require that the flex layout used below has a `spaceitems` prop that sets the space to `sm` or smaller. But I don't think that works with the current font size at 2xl. The other alternative is to replace the Split layout with a flex layout where we have more control over the space between flex items, so we could have a `lg` space between the 2 metrics and a `md` space between the icon and text. 🤷‍♀️  

![image](https://user-images.githubusercontent.com/21063328/224843380-d447e5e2-4089-47a8-8b0b-2b917f84eefe.png)